src/TimeSeries.svelte

```diff
@@ -52,8 +54,10 @@
   const margin = { top: 10, bottom: 30, left: 0, right: 30 };
 
   function getLegendComponentText(): string {
-    if (selectedRegion.sub_region_3) {
-      return "";
+    if (selectedCountryMetadata.countryCode != "IE") {
```

Comment:
@Jelena0921 this looks like it came from your merge...

Reply:
If I remember correctly, you added this as you were getting weird errors for Ireland. If you still get errors when removing this check, please use selectedCountryMetadata.displayLevels includes 3 instead. But overall it's weird that it throws that error, as selectedRegion should still have sub_region_3, as it's required in the class, but it would just be empty for Ireland.